test/circle.cooltt

```diff
@@ -0,0 +1,17 @@
+def Ω1s1 : type =
+  pathd {_ => circle} base base
+
+def loopn : nat -> Ω1s1 =
+  elim [
+  | zero => _ => base
+  | suc {n => loopn} =>
+    i =>
+    hcom circle 0 1 {∂ i} {k _ =>
+      [ k=0 => loopn i
+      | i=0 => base
+      | i=1 => loop k
+      ]
+    }
+  ]
+
+normalize {loopn 13} -- the biggest one it can compute before Stack Overflow lol
```

Comment:
😎

Reply:
Oh lol I forgot to commit my test file but you pretty much replicated it

Reply:
Hahaha

Reply:
I was like "Wow Evan is so amazing, he even managed to code this without any examples ??"